dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -199,7 +200,14 @@ const SignInGateSelectorDefault = ({
 	pageId,
 	idUrl = 'https://profile.theguardian.com',
 	switches,
+	contributionsServiceUrl,
 }: Props) => {
+	// comment group: auxia-prototype-e55a86ef
+	// The following (useless) instruction only exists to avoid linting error
```

Comment:
appreciate this bit is only temporary but the standard way of doing this would be a `// eslint-disable-next-line` comment

Reply:
True, but the comment is for future self (so that I remember "why" I did that), not for the linter :)